docs/app/pages/components/sections/tables/column-sorting/column-sorting.component.html

```diff
@@ -62,12 +60,25 @@
             <td>Defines the sort key for the column.</td>
         </tr>
 
+        <tr>
+            <td><b>@Input</b> singleSort</td>
```

Comment:
I think this should be moved to the Column Sorting directive rather than the component, otherwise they will have to add this on each column sorter they use, rather than just once on the directive.